src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/wwwroot/css/app.css

```diff
@@ -1,3 +1,4 @@
+/*#if (SampleContent)*/
```

Comment:
This if block doesn't work. For empty and non-empty blazor wasm template this comment stays in plays.
I checked for Blazor Web template [app.css](https://github.com/dotnet/aspnetcore/blob/main/src/ProjectTemplates/Web.ProjectTemplates/content/BlazorWeb-CSharp/BlazorWeb-CSharp/wwwroot/app.css#L1) and it works there.

@SteveSandersonMS, do you know why this happens?

Reply:
@MackinnonBuck maybe you know?

Reply:
I think it's because of this block in `template.json`:

```json
      "copyOnly": [
        "**/wwwroot/css/**"
      ],
```

You'll need to change it to something like:

```json
      "copyOnly": [
        "**/wwwroot/css/bootstrap/**"
      ],
```

... so that `app.css` is not included in `copyOnly`.